backend/database/bespoke.ts

```diff
@@ -132,12 +132,30 @@ export class BespokeQueries {
     `);
   }
 
+  static async itemHasTransactions(
```

Comment:
Redundant function, if it's being used elsewhere (other than this PR) rewrite using `db.count`

Bespoke is meant only for queries that aren't possible using the standard `db` methods.

Reply:
corrected